app/views/users/show.html.erb

```diff
@@ -1 +1,34 @@
-<p>Here is a user with this id</p>
+<section id='user_box'>
+  <img class='image' src=<%= @user.photo %> alt='profile picture' />
+  <div class='user_name'>
+    <h1 class='name'><%= @user.name %></h1>
+    <p class='posts_counter'>Number of posts: <%= @user.posts_counter%></p>
+  </div>
+</section>
+<section class='bio'>
+  <h4>Bio</h4>
+  <p><%= @user.bio %></p>
+</section>
+
+<section id='posts'>
+  <% @user.three_most_recent_posts.each do |recent_post| %>
+    <article id='post'>
+      <h4><%= recent_post.title %></h4>
+      <div class='texts'>
+        <p class='text'><%= recent_post.text %></p>
+        <div class='comlik'>
+          <p>
+            <span class='comments'>Comments: <%= recent_post.comments_counter %></span>,
+            <span class='likes'>Likes: <%= recent_post.likes_counter %></span>
+          </p>
+        </div>
+      </div>
+    </article>
+  <% end %>
+</section>
+
+<div id='allposts'>
+  <%= link_to "/users/#{@user.id}/posts" do %>
+    <button>See all post</button>
+  <% end %>
+</div>
```

Comment:
Kindly consider creating a button or link for creating posts  and comments for the user since there is currently no link in the app for that. Moreover, kindly ensure it is working as it's suppose to be, because I tried creating a post by typing the link in the browser directly and I got an error as shown below. :pray: 

![Screenshot from 2023-03-11 19-13-52](https://user-images.githubusercontent.com/65233860/224508019-d4024bea-8114-4798-aa84-10ad869b5d80.png)

Reply:
Thanks @jerryowusu , but can I know what link you used, as it was working here perfectly

Reply:
@Ibnballo1, since there is no link to the post form in the app, I added /new to the user' show page of the first user, which displayed the form in the app, after which I typed in the title and text inputs.